### DISCUSSION TITLE
CLAP host validator

### DISCUSSION THREAD

**Discussion by xris1658** [free-audio/clap#301] [Ideas]
Is there a CLAP host validator which checks if the host does its things correctly (kinda like [VST3 Host Checker](https://github.com/steinbergmedia/vst3_public_sdk/tree/master/samples/vst/hostchecker))?

**Comment by abique** [comment]
If you're implementing you're plugin using clap helpers, the glue layer already contains an extensive contract checking which can be turned on/off via template parameters.
